webapp/src/shared/hooks/UseMediaQuery.tsx

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react';
+
+export const useMediaQuery = (screenResolutionInPx: number) => {
```

Comment:
This could be avoided with some of the suggestions I made above

Reply:
Why should we, I wonder, avoid this.

Reply:
Apparently, we did not need it. I think if I was still trying to align it to the top of the header, I would still not know how to work around it. But as soon as I removed, this wasn't a problem anymore.